src/opencl_common.h

```diff
@@ -152,6 +152,11 @@ extern int device_info[MAX_GPU_DEVICES];
 #define DUR_CONFIG_NAME         "_MaxDuration"
 #define FALSE               0
 
+#define CL_SOFT_ERROR           1
+#define CL_HARD_ERROR           2
+#define CL_IGNORE_ERROR         3
+#define CL_MAX_BUFFERS         32
+
```

Comment:
All of these use cases must use `CL_HARD_ERROR`. Remove this.

Reply:
I was thinking the same (before I saw your comment) - we must not ignore this kind of errors, or we'd crash on a dependent call next. So no reason to support ignoring them.